src/examples/zkapps/dex/dex-with-actions.ts

```diff
@@ -71,7 +79,7 @@ class Dex extends SmartContract {
   }
 
   @method createAccount() {
-    this.token.mint({ address: this.sender, amount: UInt64.from(0) });
+    this.internal.mint({ address: this.sender, amount: UInt64.from(0) });
```

Comment:
`this.internal` is a great way to communicate usage within smart contract methods, awesome!